tests/test_unravel.py

```diff
@@ -71,7 +73,15 @@ def test_unravel_plane():
                                    [  32.2554512 ,   56.86440277,   49.46508026],
                                    [  -4.24387503,   47.21520996,   52.44573212]])
 
-def test_unravel_plane():
```

Comment:
The name test_unravel_plane was used for 2 unit tests so the previous one wasn't actually run.